_posts/2017-11-15-containerised-application-deployment-solutions.md

```diff
@@ -0,0 +1,85 @@
+
+
+#### Google Cloud
+
+My experience with [Google Cloud](https://cloud.google.com) was a lot shorter than anticipated.
+<br/>
+As a non-business European user, Google Cloud is not available:
+
+> If you are located in the European Union and the sole purpose for which you want to use Google Cloud Platform services has no potential economic benefit you should not use the service. If you have already started using Google Cloud Platform, you should discontinue using the service. See Create, modify, or close your billing account to learn how to disable billing on your projects.
+> [Google Cloud](https://cloud.google.com/free/docs/frequently-asked-questions)
+
+Googling the topic a bit, it looks like it has something to do with taxes. Google Cloud needs a VAT number so you must be a business to use Google Cloud Platform.
+<br/>
+
+Oh well... Next!
+
+
+#### Amazon Web Service (AWS)
+
+[AWS](https://aws.amazon.com) offers free accounts like Azure so it is really great for testing. It also provides many deployment solutions for containerised applications, here are the ones I found:
+
+* [EC2 Container Service (ECS)](https://aws.amazon.com/ecs)
+<br/>ECS is similar to AKS (I think ECS may have existed before AKS so I shall probably say "Microsoft Azure offers a service similar to ECS" :)).
+<br/>I actually tested this one and was able to deploy my application to AWS but I didn't really enjoy the experience. I followed this tutorial [here](http://docs.aws.amazon.com/AmazonECS/latest/developerguide/ECS_GetStarted.html) and it took me quite some time to create all the things that were required: cluster, service, task, IAM, etc... Having spent so much time looking at deployment solutions I realise now that I shouln't even have considered this service. It is for applications way more complex than mine!
+
+* [Elastic Beanstalk](https://aws.amazon.com/elasticbeanstalk)
+<br/>I somehow missed this solution, one of my friends mentioned it to me.
+<br/>Elastic Beanstalk is a simple service to deploy and scale applications. I haven't looked into it yet but it looks like it supports Docker and it could a good candidate for my use case. However, I must admit I am not a big fan of Amazon documentation.
+
+
+#### IBM Bluemix
+
+As I was searching for new alternatives to deploy my application I came across this great [video](https://www.youtube.com/watch?v=TfCj2qOXb1g) by David Barnes, IBM technology evangelist. It offered a very simple solution to deploy containerised applications, I was very excited about it. However, I quickly realised that the service was being deprecated [:'(](https://console.bluemix.net/docs/containers/cs_classic.html).
+<br/>
+IBM new alternative is a service similar to Microsoft AKS and Amazon ECS called [IBM Cloud Container Service](https://www.ibm.com/cloud/container-service). As I said before, this type of services are great but add too much complexity to my simple use case.
```

Comment:
_these_ types of _service_ are great

~to~ for my simple use case

Reply:
_IBM's_ new alternative

Reply:
_these_ types of _service_ are great => I mean _this type of service is great_
👍 
👍